source/uwp/Renderer/lib/AdaptiveChoiceSetInputRenderer.h

```diff
@@ -28,6 +26,16 @@ namespace AdaptiveNamespace
                                 _In_ ABI::AdaptiveNamespace::IAdaptiveActionParserRegistration* actionParsers,
                                 _In_ ABI::Windows::Foundation::Collections::IVector<ABI::AdaptiveNamespace::AdaptiveWarning*>* adaptiveWarnings,
                                 _COM_Outptr_ ABI::AdaptiveNamespace::IAdaptiveCardElement** element) noexcept override;
+
+    private:
+        HRESULT _BuildExpandedChoiceSetInput(_In_ ABI::AdaptiveNamespace::IAdaptiveRenderContext* renderContext,
```

Comment:
>_ [](start = 16, length = 1)

why do these methods start with _ ? Is it to avoid build errors?

Reply:
old habits die hard. it's to denote them as private methods. I'll fix since we don't use this convention here :)

---
In reply to: [326334977](https://github.com/microsoft/AdaptiveCards/pull/3457#discussion_r326334977) [](ancestors = 326334977)